recipes/trexio/meta.yaml

```
@@ -0,0 +1,58 @@
+
+{% set name = "trexio" %}
+{% set version = "2.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/TREX-CoE/trexio/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 232866c943b98fa8a42d34b55e940f7501634eb5bd426555ba970f5c09775e83
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - autoconf  
+    - automake  
+    - libtool   
+    - make      
+    - gnuconfig 
+    - pkg-config
+
+  host:
+    - hdf5
+    # the dependencies below are not required by TREXIO but they get exported from HDF5
+    # and if they are not listed here than the `conda-build --error-overlinking` fails
+    - zlib    
+    - openssl 
+    - libcurl 
+
+  run:
+    - hdf5 
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libtrexio.a  # [unix]
+
+about:
+  home: https://github.com/TREX-CoE/trexio
+  license: BSD-3-Clause
+  license_family: BSD 
+  license_file: COPYING
```

```suggestion
  license_file: LICENSE
```
Copying is a symbolic link to LICENSE. Add LICENSE directly?

---

Right, good idea! Done.

---

In fact, I just realized that `COPYING` is a symbolic link only when building from the GitHub repo clone. In the distribution tarball produced by Autotools (`make dist`) it appears as a text file, so it's fine. The `LICENSE` is not shipped with the distribution tarball. So we can keep it for now.